app/views/admin/users/show.html.erb

```diff
@@ -13,7 +13,7 @@
 
   <%= render partial: "shared/search_bar", locals: {url: admin_user_path, query: params[:search]} %>
 
-  <table id="users" class="table table-hover table-condensed">
+  <table id="users" class="table table-hover table-condensed" ng-controller="ProjectRolesCtrl" ng-init="loadProjectRoles()">
```

Comment:
As per the docs: https://docs.angularjs.org/api/ng/directive/ngInit
Just move this initialization into the controller itself. 

I.e., just call the loadProjectRoles() function at the bottom of the controller source itself.